Build fails on Humble (Ubuntu 22.04) – multiple dependencies unresolved with install.sh

Hello,

I have been trying to set up arena-rosnav on ROS 2 Humble (Ubuntu 22.04) by strictly following the provided install.sh script. Despite multiple clean installations and repeated attempts, I have not been able to get the build to finish successfully using only the installer.

Initially, the build failed with:
`Summary: 0 packages finished [1.19s]
  6 packages failed: actuator_msgs cv_bridge gps_msgs image_geometry pcl_msgs vision_msgs
  6 packages not processed`

When I attempted to manually install some of these dependencies and re-run the build, I encountered further errors:
`Summary: 0 packages finished [2.36s]
12 packages failed: arena_bringup arena_rclpy_mixins gps_tools gpsd_client opencv_tests plan_msgs rviz_utils task_generator_msgs testing unity_msgs vision_msgs_rviz_plugins vision_opencv`

`--- stderr: vision_msgs_rviz_plugins
CMake Error at CMakeLists.txt:62 (add_library):
  Target "vision_msgs_rviz_plugins" links to target "yaml-cpp::yaml-cpp" but
  the target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?`

It seems that several key dependencies (both upstream packages like cv_bridge, image_geometry, vision_opencv and arena-related packages such as arena_bringup, task_generator_msgs) are not resolving or building properly in Humble through the current workflow.

Could you please confirm if Humble on Ubuntu 22.04 is officially supported, and if there are additional manual steps required beyond the provided install.sh? Any guidance on resolving these dependency issues would be greatly appreciated.

--- a/installers/install.sh
+++ b/installers/install.sh
@@ -166,7 +166,7 @@ if [ ! -d src/deps ] ; then
     git clone --filter=tree:0 --depth 1 https://github.com/ros-perception/pcl_msgs.git -b ros2
     git clone --filter=tree:0 --depth 1 https://github.com/rudislabs/actuator_msgs.git
     git clone --filter=tree:0 --depth 1 https://github.com/swri-robotics/gps_umd.git -b ros2-devel
-    git clone --filter=tree:0 --depth 1 https://github.com/ros-perception/vision_msgs.git -b ros2
+    git clone --filter=tree:0 --depth 1 https://github.com/ros-perception/vision_msgs.git -b humble
     git clone --filter=tree:0 --depth 1 https://github.com/ros-perception/vision_opencv.git -b humble
   popd
 fi

